### PR TITLE
Updating destroy plan behavior

### PR DESCRIPTION
**Pull request by sudomateo**
Terraform Enterprise v201912-1 introduced a change how destroy plans behave when auto apply is enabled on the workspace. This pull request updates the documentation to reflect that new behavior.

### CODE REVIEW

File: content/source/docs/cloud/workspaces/settings.html.md [hunk 1]
```diff
@@ -56,9 +56,8 @@ To disable remote execution for a workspace, change its execution mode to "Local
 
 Whether or not Terraform Cloud should automatically apply a successful Terraform plan. If you choose manual apply, an operator must confirm a successful plan and choose to apply it.
 
-Auto-apply has a few exceptions:
+Auto-apply has the following exceptions:
```
Comment (justinretzolk) [inline content/source/docs/cloud/workspaces/settings.html.md]: Perhaps this should be "exception" given that there's now only one?

File: content/source/docs/cloud/api/run.html.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ Parameter | Description
 ----------|------------
 `run_id`  | The run ID to apply
 
-Applies a run that is paused waiting for confirmation after a plan. This includes runs in the "needs confirmation" and "policy checked" states. This action is only required for runs that can't be auto-applied. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace, the plan is not a destroy plan, and the plan was not queued by a user without write permissions.)
+Applies a run that is paused waiting for confirmation after a plan. This includes runs in the "needs confirmation" and "policy checked" states. This action is only required for runs that can't be auto-applied. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace and the plan was not queued by a user without write permissions.)
```
Comment (nfagerlund) [inline content/source/docs/cloud/api/run.html.md]:
```suggestion
Applies a run that is paused waiting for confirmation after a plan. This includes runs in the "needs confirmation" and "policy checked" states. This action is only required for runs that can't be auto-applied. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace and the plan was queued by a new VCS commit or by a user with write permissions.)
```

In retrospect, that double-negative is confusing, so let's tweak that.

File: content/source/docs/cloud/run/index.html.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ In more abstract terms, runs can be initiated by VCS webhooks, the manual "Queue
 
 ## Plans and Applies
 
-Terraform Cloud enforces Terraform's division between _plan_ and _apply_ operations. It always plans first, saves the plan's output, and uses that output for the apply. In the default configuration, it waits for user approval before running an apply, but you can configure workspaces to [automatically apply](../workspaces/settings.html#auto-apply-and-manual-apply) successful plans. (Some plans can't be auto-applied, like destroy plans or plans queued by users without write permissions.)
+Terraform Cloud enforces Terraform's division between _plan_ and _apply_ operations. It always plans first, saves the plan's output, and uses that output for the apply. In the default configuration, it waits for user approval before running an apply, but you can configure workspaces to [automatically apply](../workspaces/settings.html#auto-apply-and-manual-apply) successful plans. (Some plans can't be auto-applied, like plans queued by users without write permissions.)
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/index.html.md]:
```suggestion
Terraform Cloud enforces Terraform's division between _plan_ and _apply_ operations. It always plans first, saves the plan's output, and uses that output for the apply. In the default configuration, it waits for user approval before running an apply, but you can configure workspaces to [automatically apply](../workspaces/settings.html#auto-apply-and-manual-apply) successful plans. (Some plans can't be auto-applied, like plans queued by [run triggers](../workspaces/run-triggers.html) or by users without write permissions.)
```

File: content/source/docs/cloud/run/states.html.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ _Leaving this stage:_
 - If the plan succeeded and requires changes:
     - If cost estimation is enabled, the run proceeds automatically to the cost estimation stage.
     - If cost estimation is disabled and [Sentinel policies][] are enabled, the run proceeds automatically to the policy check stage.
-    - If there are no Sentinel policies and the plan can be auto-applied, the run proceeds automatically to the apply stage. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace, the plan is not a destroy plan, and the plan was not queued by a user without write permissions.)
+    - If there are no Sentinel policies and the plan can be auto-applied, the run proceeds automatically to the apply stage. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace and the plan was not queued by a user without write permissions.)
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/states.html.md]:
```suggestion
    - If there are no Sentinel policies and the plan can be auto-applied, the run proceeds automatically to the apply stage. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace and the plan was queued by a new VCS commit or by a user with write permissions.)
```

File: content/source/docs/cloud/run/states.html.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ _Leaving this stage:_
     - If a member of the owners team overrides the failed policy, the run proceeds to the **Policy Checked** state.
     - If an owner or a user with write access discards the run, the run skips to completion (**Discarded** state).
 - If the run reaches the **Policy Checked** state (no mandatory policies failed, or soft-mandatory policies were overridden):
-    - If the plan can be auto-applied, the run proceeds automatically to the apply stage. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace, the plan is not a destroy plan, and the plan was not queued by a user without write permissions.)
+    - If the plan can be auto-applied, the run proceeds automatically to the apply stage. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace and the plan was not queued by a user without write permissions.)
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/states.html.md]:
```suggestion
    - If the plan can be auto-applied, the run proceeds automatically to the apply stage. (Plans can be auto-applied if the auto-apply setting is enabled on the workspace and the plan was queued by a new VCS commit or by a user with write permissions.)
```

File: content/source/docs/cloud/run/ui.html.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ Note that once the plan stage is completed, until you apply or discard a plan, T
 
 ### Auto apply
 
-If you would rather automatically apply plans that don't have errors, you can [enable auto apply](../workspaces/settings.html#auto-apply-and-manual-apply) on the workspace's "General Settings" page. (Some plans can't be auto-applied, like destroy plans or plans queued by users without write permissions.)
+If you would rather automatically apply plans that don't have errors, you can [enable auto apply](../workspaces/settings.html#auto-apply-and-manual-apply) on the workspace's "General Settings" page. (Some plans can't be auto-applied, like plans queued by users without write permissions.)
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/ui.html.md]:
```suggestion
If you would rather automatically apply plans that don't have errors, you can [enable auto apply](../workspaces/settings.html#auto-apply-and-manual-apply) on the workspace's "General Settings" page. (Some plans can't be auto-applied, like plans queued by [run triggers](../workspaces/run-triggers.html) or by users without write permissions.)
```

File: content/source/docs/cloud/workspaces/settings.html.md [hunk 7]
```diff
@@ -56,9 +56,8 @@ To disable remote execution for a workspace, change its execution mode to "Local
 
 Whether or not Terraform Cloud should automatically apply a successful Terraform plan. If you choose manual apply, an operator must confirm a successful plan and choose to apply it.
 
-Auto-apply has a few exceptions:
+Auto-apply has the following exception:
 
-- [Destroy plans](#destruction-and-deletion) must always be manually applied.
 - Plans queued by users with [plan permissions](../users-teams-organizations/permissions.html#plan) must be approved by a user with write or admin permissions.
```
Comment (nfagerlund) [inline content/source/docs/cloud/workspaces/settings.html.md]:
```suggestion
- Plans queued by users with [plan permissions](../users-teams-organizations/permissions.html#plan) must be approved by a user with write or admin permissions.
- Plans queued due to [run triggers](../workspaces/run-triggers.html) from another workspace must always be manually applied.
```

**Pull request merged**